public/style.css

```diff
@@ -0,0 +1,50 @@
+body {
+    font-family: Arial;
+    margin-left: 5%;
+    margin-top: 20%;
+  }
+  
+  * {
+    box-sizing: border-box;
+  }
+  
+  form.search input[type=text] {
+    padding: 10px;
+    font-size: 17px;
+    border: 1px solid grey;
+    float: left;
+    width: 80%;
+    background: #f1f1f1;
+  }
+  
+  form.search button {
+    float: left;
```

Comment:
Why are you using float? Try using float or grid.

Reply:
@jema28  no need for float, I remove it